appendix/basic_commands.tex

```diff
@@ -1498,24 +1498,24 @@ \subsection{GET}
 \subsection{GET\#}
 
 The {\ttfamily GET\#} statement reads data from a specified logical file one
-character at a time.  It is identical to the {\ttfamily GET} statement except
-that it requires a first argument specifies a logical file identifier.  The
+character at a time.  It is comparable ***SIMILAR?*** to the {\ttfamily GET} statement although 
+it requires a first argument that specifies a logical file identifier.  The
 logical file must first be opened with the {\ttfamily OPEN} statement.  The
 remaining arguments work just the same as with the {\ttfamily GET} statement.\\
 
 Example:\\
 
 \codeblock{
-	10 OPEN 1,1,0,"FILENAME"  REM OPEN TAPE DRIVE FILE\\
-	20 GET\#1,A\$  REM READ A CHARACTER FROM THE TAPE\\
+	10 OPEN 1,1,0,"FILENAME" : REM OPEN TAPE DRIVE FILE\\
+	20 GET\#1,A\$ : REM READ A CHARACTER FROM THE TAPE\\
 }
 
 \note{
 
 	It is a popular convention when using statements and commands that end in
 	{\ttfamily \#} to place the logical file identifier directly after the
-	command with no space.  So instead of {\ttfamily GET\# 1,A\$} this example
-	shows {\ttfamily GET\#1,A\$}.
+	command with no space.  So instead of\\
+	{\ttfamily GET\# 1,A\$} this example shows {\ttfamily GET\#1,A\$}.
```

Comment:
This is intended as an inline code example, and not its own code block.  As such, it's fine if the line wraps in the middle.  If we ever do decide we don't like this convention, then we won't fix it by inserting newlines.  We will fix it by creating (or finding) a text element that prevents wrapping, and putting the inline code examples inside it. Adding newlines to arbitrary places will cause huge issues later if we change fonts, font sizes, margins, or any other stylistic elements.

Reply:
The added forced new line because in output it currently split GET(space) and the #1 over 2 lines, so you couldn't see the "space" effect (difference).

Reply:
Even so, this is a very dangerous way to force a new line.  Imagine if we reword the sentence above, so that the only word left on the line is the preceding "of".    The same could happen if we choose a new font or margin size.  Basically, we can never assume that a newline inserted for nicer formatting _today_ will result in better formatting _later_.

For now, I don't really care about line breaks in the middle of inline code snippets, but if I ever do, we need to come up with a much better solution than just inserting newlines.  Probably this: https://stackoverflow.com/questions/1012799/latex-prevent-line-break-in-a-span-of-text